lib/ruby_lsp/executor.rb

```diff
@@ -523,7 +523,14 @@ def check_formatter_is_available
       # Syntax Tree will always be available via Ruby LSP so we don't need to check for it.
       return unless @store.formatter == "rubocop"
 
-      unless defined?(Support::RuboCopFormattingRunner)
+      rubocop_available = begin
+        require "rubocop"
+        true
```

Comment:
Needed because if something else has already called `require "rubocop"` then the second require will return false.